packages/visualizations-react/stories/Chart/StudioLayouts/AreaChart.stories.tsx

```diff
@@ -28,7 +28,6 @@ const AreaTitleAxisGridDotsArgs: Props<DataFrame, ChartOptions> = {
     options: {
         labelColumn: 'x',
         source: defaultSource,
-        ariaLabel: 'Area chart with title, axis, grid and dots',
```

Comment:
As `StudioChoropleth.stories.tsx` has a description in `options`. 
Maybe we should add `description` in all Chart studio story too.

Reply:
I added it in only one of each, just to make sure they are hidden (Chromatic would tell us if a story suddenly had text appearing). I don't think we need to add it everywhere, just that we have at least one of each case (with and without)